code/modules/shuttle/emergency.dm

```diff
@@ -797,6 +797,27 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/item/storage/pod, 32)
 	new /obj/item/bodybag/environmental(src)
 	new /obj/item/bodybag/environmental(src)
 
+/obj/item/storage/box/emergency_spacesuit
+	name = "emergency space suit case"
+	desc =  "A small case containing an emergency space suit and helmet."
+	icon = 'icons/obj/storage/storage.dmi'
+	icon_state = "emergency_spacesuit_box"
+	illustration = null
+
+/obj/item/storage/box/emergency_spacesuit/Initialize(mapload)
+	. = ..()
+	w_class = WEIGHT_CLASS_BULKY
```

Comment:
this can be set on the type itself, doesn't need to be set on init

Reply:
oh yeah my bad